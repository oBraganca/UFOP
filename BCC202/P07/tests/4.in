29 22 28 20 24 18 10 17 2 3 7 23 12 9 16 25 15 19 8 21 1 11 6 13 14 26 27 4 29 5
135 107 103 51 3 83 121 119 90 32 45 29 25 84 21 88 44 82 41 120 115 33 80 125 46 124 7 93 73 111 118 76 10 92 24 50 34 14 63 56 74 57 5 23 113 116 79 71 17 42 91 11 110 68 123 19 20 112 31 130 54 129 59 81 37 98 66 75 4 86 49 15 89 133 85 105 12 8 70 13 102 100 126 96 135 6 114 1 43 72 38 65 94 87 36 52 104 64 26 77 128 30 95 109 78 117 131 60 35 47 48 134 106 62 67 132 18 58 97 39 40 127 27 99 53 101 22 122 108 9 69 16 2 55 61 28
142 16 122 141 86 66 137 56 68 67 7 135 52 63 87 140 32 138 28 109 127 121 5 51 120 1 47 49 69 134 77 37 90 107 118 61 12 27 24 85 25 95 125 89 71 31 131 114 74 93 110 50 20 54 39 96 42 79 23 26 35 70 60 112 57 10 9 72 3 117 45 119 139 13 2 48 36 78 64 80 97 115 101 133 11 128 99 18 103 130 19 129 58 92 14 126 82 17 40 46 33 100 59 132 21 108 104 91 102 53 30 15 41 83 94 8 105 55 62 29 142 88 111 84 44 4 124 81 113 6 136 75 22 116 106 98 43 73 38 123 65 34 76
152 39 76 40 138 35 10 100 54 60 47 145 104 106 118 130 21 33 22 12 105 116 43 7 18 25 9 110 146 142 141 4 147 30 94 107 89 13 14 99 96 58 127 73 64 61 125 1 29 117 68 123 85 17 132 149 16 36 56 133 115 129 113 97 122 63 102 143 137 84 67 24 66 37 65 95 126 53 23 80 124 131 59 79 62 119 81 55 120 109 42 128 103 75 6 101 112 45 139 27 51 151 135 93 70 82 3 83 46 28 144 91 148 87 111 38 52 57 74 152 8 48 150 90 108 72 26 136 98 50 78 69 5 31 34 32 11 19 20 140 134 92 77 88 41 2 71 49 44 114 15 121 86
17 2 9 4 14 3 11 10 17 6 1 8 7 13 16 15 12 5
0
