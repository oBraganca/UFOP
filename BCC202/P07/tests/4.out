Marcelo
Marcelo
Carlos
Marcelo
Carlos
